Core/src/main/java/org/tribuo/Example.java

```diff
@@ -47,8 +47,14 @@
 public abstract class Example<T extends Output<T>> implements Iterable<Feature>, Serializable {
     private static final long serialVersionUID = 1L;
 
+    /**
+     * The default initial size of the metadata map.
+     */
     protected static final int DEFAULT_METADATA_SIZE = 2;
 
+    /**
+     * The default weight.
+     */
     public static final float DEFAULT_WEIGHT = 1.0f;
```

Comment:
What's the coding standard for numbers like 1 and 0. If I recall correctly, just using 1 and 0 will get you the 1 or 0 of the right type. Having it be explicit is probably better?

Reply:
I tend to put the specifier if it's a long, or if it's a float, and will put `d` on things if they are integers assigned to longs, but I usually prefer to put the decimal place in that case.